doc/APIGuide.md

```diff
@@ -57,28 +57,28 @@ model.getMassCenter(state); // Need to know pose of model.
 ## Object {#object}
 
-[Object](@ref OpenSim::Object) is the base class for all OpenSim Components, general containers like Set, and Tools that require serialization - that is having their attributes written out to file. Objects user Properties to serialize (write to file) and deserialize (read from file) their attributes, which are parameters and settings that define the behavior of the Object.
+[Object](@ref OpenSim::Object) is the base class for all OpenSim Components, general containers like Set, and Tools that require serialization - that is having their attributes written out to file. Objects use Properties to serialize (write to file) and deserialize (read from file) their attributes, which are parameters and settings that define the behavior of the Object.
 
 ## Component {#component}
 
 A [Component](@ref OpenSim::Component) is the basic “unit” of modelling and computing in OpenSim. It is our abstraction to capture a fundamental unit of computing and/or modeling used to compose a Model and form computational system of equations. It is the base class that provides the supporting infrastructure for building up a hierarchically structured model with interconnections and numerous inputs and outputs, but is represented by a single computationally efficient (mathematical/numerical) system. That computational system is a SimTK::MultibodySystem and it is the computer implementation (think system of equations) of the multibody kinematics and dynamics plus any additional dynamics (ordinary differential equations, e.g. muscle activation dynamics) that need to be solved simultaneously. Any conceptual element that has to compute something typically requires the System and must be a Component in OpenSim. Component also provides the resources for reading and writing (also known as serializing, which it inherits from Object) the attributes/parameters that define the behavior of the component and adding in its dynamics (equations). The serializable attributes of a Component are called properties in OpenSim. A Component has the following user-facing attributes:
 
-1. Properties: Constant parameters (e.g., a Body’s mass).
+1. Properties: Built-in parameters (e.g., a Body’s mass).
 2. Sockets to other components that it depends on (e.g., a Joint connects two Frames).
 3. Input quantities that the component needs to do its job (e.g., a metabolics calculator component could have a “muscle power” input).
 4. Output quantities that component can compute, and that can be used to satisfy other components’ Inputs.
-5. Subcomponents: A Component can rely on other (internal) components to do its job (e.g., a Muscle component can use subcomponents for activation and fiber-contraction dynamics).    
+5. (Internal) Subcomponents: A Component can rely on other (internal) components to do its job (e.g., a Muscle component can use subcomponents for activation and fiber-contraction dynamics). These are not user facing and are of interest only to the component writer.    
```

Comment:
There are multiple types of subcomponents, some of which can be user-facing. It's the "member subcomponents" that are partially not user-facing.

Reply:
I understood the original text to refer to Subcomponents as "internal" components which contradicts the "user-facing" statement on top. Did I misread the statement?

Reply:
The  "(Internal)"'s can be removed. Components are not "user-facing". Only properties are user-specified.
There are different types of subcomponents: properties and members. A Component's properties that are components are subcomponents. A component can also have data members that are Components, which make them "internal" in the sense that their properties are not accessible by the user.

Reply:
I removed all "internal" references.